Code/Framework/AzFramework/AzFramework/Components/TransformComponent.h

```diff
@@ -186,5 +187,7 @@ namespace AzFramework
         bool m_parentActive = false; ///< Keeps track of the state of the parent entity.
         bool m_onNewParentKeepWorldTM = true; ///< If set, recompute localTM instead of worldTM when parent becomes active.
         bool m_isStatic = false; ///< If true, the transform is static and doesn't move while entity is active.
+        /// Behavior for this entity's transform when its parent's transform changes.
```

Comment:
suggestion: Might be worth mentioning something like 'see OnParentChangedBehavior definition for definitions of behavior types' (maybe overkill but could be useful to give readers a hint)

Reply:
That does feel a bit overkill to me tbh. The first thing I would do reading the code is F12 on the type, which takes you straight to the definitions. Also I think they're fairly clear from the names.